src/components/App.js

```diff
@@ -3,11 +3,15 @@ import Container from 'components/Container/Container'
 import Input from 'components/Input/Input'
 import List from 'components/List/List'
 import ListItem from 'components/ListItem/ListItem'
+import Button from 'components/Button/Button'
 
 const App = () => {
   return (
     <Container>
-      <Input />
+      <Container>
+        <Input />
+        <Button>Agregar</Button>
```

Comment:
Fijate que te faltó agregarle la clase 

```suggestion
        <Button className="button">Agregar</Button>
```